kaizen/cli.py

```
@@ -0,0 +1,125 @@
+
+@cli.command()
+@click.argument('test_file', type=click.Path(exists=True))
+def run_test(test_file: str):
+    """Run a test file and display results."""
+    try:
+        with open(test_file, 'r') as f:
+            test_config = yaml.safe_load(f)
+        
+        # Create test runner
+        runner = TestRunner()
+        logger = TestLogger(test_config.get('name', 'Unnamed Test'))
```

Ruff (0.11.9)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)